montrek-software/montrek
# 378 media root in test case

# How to test

Run some test (e.g. `python manage.py test file_upload`) Make sure no files are written to `uploads/`
